neqo-transport/src/recv_stream.rs

```diff
@@ -336,7 +336,7 @@ impl RecvStreamState {
 
 /// Implement a QUIC receive stream.
 #[derive(Debug)]
-pub struct RecvStream {
+pub(crate) struct RecvStream {
```

Comment:
We don't `pub use recv_stream`, so this extra annotation shouldn't be necessary, right?

Reply:
see prev comment, we used to need this. Life moves pretty fast, I guess. They're gone now.